### PR TITLE
Imported new table for reservations, with filtering options

### PR DESCRIPTION
**Pull request by KrzBan**
image

### CODE REVIEW

**Comment by KrzBan** [comment]
image

**Comment by KrzBan** [comment]
image

File: frontend/src/components/ReservationsTable.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+          accessorKey: 'end_time',
+          header: 'Czas Zakończenia',
+        },
+        {
+          accessorFn: (row: any) =>{
+            const val = row.status;
+            switch(val){ 
+              case "open": return "Otwarta"; break;
+              case "accepted": return "Zaakceptowana"; break;
+              case "declined": return "Odrzucona"; break;
```
Comment (Denwiliusz27) [inline frontend/src/components/ReservationsTable.tsx:45]: "break" should be deleted

File: frontend/src/components/ReservationsTable.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
+import React, { useMemo } from 'react';
+import MaterialReactTable, { type MRT_ColumnDef } from 'material-react-table';
```
Comment (Denwiliusz27) [inline frontend/src/components/ReservationsTable.tsx:2]: delete unused imports

**Review by Denwiliusz27 — APPROVED** [review]
lgtm

**Pull request merged**